Fail to run marshalling example

I have pulled the latest git master (rev 11c75f6b) atm, and I just ran `cargo run --example marshalling`. It fails with the following error:
```
Compiling gluon v0.8.0 (file:///Users/jelle/Sync/Projects/gluon)
error[E0433]: failed to resolve. Could not find `ser` in `api`
  --> examples/marshalling.rs:44:14
   |
44 |         api::ser::Ser(self).push(thread, context)
   |              ^^^ Could not find `ser` in `api`

error[E0433]: failed to resolve. Could not find `de` in `api`
  --> examples/marshalling.rs:50:14
   |
50 |         api::de::De::from_value(thread, value).0
   |              ^^ Could not find `de` in `api`

error[E0433]: failed to resolve. Could not find `typ` in `api`
  --> examples/marshalling.rs:67:28
   |
67 |     let enum_source = api::typ::make_source::<Enum>(&thread)?;
   |                            ^^^ Could not find `typ` in `api`

error[E0277]: the trait bound `str: std::marker::Sized` is not satisfied
  --> examples/marshalling.rs:67:9
   |
67 |     let enum_source = api::typ::make_source::<Enum>(&thread)?;
   |         ^^^^^^^^^^^ `str` does not have a constant size known at compile-time
   |
   = help: the trait `std::marker::Sized` is not implemented for `str`
   = note: all local variables must have a statically known size

error[E0277]: the trait bound `str: std::marker::Sized` is not satisfied
  --> examples/marshalling.rs:67:23
   |
67 |     let enum_source = api::typ::make_source::<Enum>(&thread)?;
   |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `str` does not have a constant size known at compile-time
   |
   = help: the trait `std::marker::Sized` is not implemented for `str`
   = note: all local variables must have a statically known size

error: aborting due to 5 previous errors

Some errors occurred: E0277, E0433.
For more information about an error, try `rustc --explain E0277`.
error: Could not compile `gluon`.

To learn more, run the command again with --verbose.
```
I've run all the tests, and they seem to work. I'm running stable Rust on macOS.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -152,9 +152,11 @@ name = "ui"
 name = "24"
 [[example]]
 name = "marshalling"
+required-features = ["serialization"]
 [[example]]
 name = "http"
 path = "examples/http/main.rs"
+required-features = ["serialization"]
 [[example]]
 name = "lisp"
 path = "examples/lisp/main.rs"

--- a/vm/src/api/de.rs
+++ b/vm/src/api/de.rs
@@ -1,4 +1,6 @@
 //! Gluon -> Rust value conversion via the `serde::Deserialize` trait
+//! 
+//! _This module requires Gluon to be built with the `serde` feature._
 
 use std::cell::RefCell;
 use std::fmt;

--- a/vm/src/api/ser.rs
+++ b/vm/src/api/ser.rs
@@ -1,4 +1,6 @@
 //! Rust -> Gluon value conversion via the `serde::Serialize` trait
+//! 
+//! _This module requires Gluon to be built with the `serde` feature._
 
 use std::fmt;
 use std::ops::{Deref, DerefMut};

--- a/vm/src/api/typ.rs
+++ b/vm/src/api/typ.rs
@@ -1,4 +1,6 @@
 //! Rust type to gluon type conversion
+//! 
+//! _This module requires Gluon to be built with the `serde` feature._
 
 use base::symbol::{Symbol, Symbols};
 use base::types::{ArcType, Field, Type, TypeCache};

